script/Ciel/Day 5/05_01_CIEL05_2_2B.txt

```diff
@@ -188,8 +188,8 @@ Even worse, that vampire has no sense of self-preservation.
 [sha:2213a0e64232ca85ac9bf051ea87683e8231ada5]{
 -- Page 373, Offset 11860.
 -- 人体だけ燃やす炎でも、吹雪でもまだマシだった。
-// Or is it saying that the blizzard is what's worse than the flames?
-Already with those flames that only burn human bodies and with this blizzard, but this is even worse.
+// i think this is right? not certain though
+Those flames that only burn human bodies and this blizzard were already bad, but this is even worse.
```

Comment:
Flames that only burn humans and that blizzard are one thing, but this is even worse.